ddtrace/tracer.py

```diff
@@ -840,6 +841,30 @@ def shutdown(self, timeout=None):
             pass
         atexit.unregister(self._atexit)
 
+    def get_correlation_ids(self):
+        # type: () -> Tuple[str, str]
+        """Retrieves the Correlation Identifiers for the current active ``Trace``.
+        This helper method can be achieved manually and should be considered
+        only a shortcut. The main reason is to abstract the current ``Tracer``
+        implementation so that these identifiers can be extracted either the
+        tracer is an OpenTracing tracer or a Datadog tracer.
+
+        OpenTracing users can still extract these values using the ``ScopeManager``
+        API, though this shortcut is a simple one-liner. The usage is:
+
+            trace_id, span_id = tracer.get_correlation_ids()
+
+        :returns: a tuple of strings containing the trace_id and span_id. Defaults to "0" if not found.
```

Comment:
same comments/suggestions as on the OT changes.

if there is another preferred way to get these ids, maybe we should just update our docs with that approach and call it a day? we need to decide if this is a long term API we want to support, if so then it isn't a "shortcut", this is "the way" to do it.

Reply:
Considering that this is quite an old helper introduced as a shortcut but hasn't changed for about 3 years (https://github.com/DataDog/dd-trace-py/pull/488), I agree it may be a good idea to introduce a better API (as suggested by @majorgreys and @brettlangdon) that can provide a larger set of log record attributes rather than just the span ID and trace ID and provide that as "the right way" for custom log users.